pkg/controller/migmigration/migrate.go

```diff
@@ -85,7 +85,7 @@ func (r *ReconcileMigMigration) migrate(migration *migapi.MigMigration) (time.Du
 		if errors.IsConflict(err) {
 			return FastReQ, nil
 		}
-		log.Trace(err)
+		log.Trace(err) // TODO - handle with liberr
 		task.fail(MigrationFailed, []string{err.Error()})
 		return task.Requeue, nil
```

Comment:
This should be:
```
- log.Trace(err)
- return task.Requeue, nil
+ return task.Requeue, liberr.Wrap(err)
```

Reply:
We want to return an err where we previously did not? Wouldn't this perhaps throw off some logic in the caller?

Reply:
We agreed that in this case, we should just leave the log.Trace where it is.